src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -534,8 +536,8 @@ void JvmtiEnvBase::destroy_jni_reference(JavaThread *thread, jobject jobj) {
 // Threads
 //
 
-jobject *
-JvmtiEnvBase::new_jobjectArray(int length, Handle *handles) {
+jthread *
+JvmtiEnvBase::new_jthreadArray(int length, Handle *handles) {
```

Comment:
Shouldn't this method need to cast the return value of `jthread*`? And potentially shouldn't all the jobject's now be jthread's?

Reply:
prims/jvmti.xml:      <definition>typedef jobject jthread;</definition>
It's essentially the same thing.  I'll add a cast so it looks good.

Reply:
jthread and jthreadGroup are typedefs to jobject in JVMTI spec.
https://docs.oracle.com/en/java/javase/11/docs/specs/jvmti.html#jthread

But I updated the code to have the more specific types.